packaging/nnstreamer.spec

```diff
@@ -102,7 +104,22 @@ pushd build
 popd
 
 pushd tests
-./testAll.sh
+timeout=600
+./testAll.sh &
+pid=$!
+while ((timeout > 0));
+do
+    sleep 1
+    if [[ ! $(ps | grep "$pid") ]]; then
+        break;
+    fi
+    (( timeout -= 1))
```

Comment:
BTW. what is the meaning of ((....)) grammar?

Reply:
It means to operate in parentheses.